web-ts/src/components/I18nProvider.tsx

```diff
@@ -70,29 +63,28 @@ export default ({ children }: PropsType) => {
       })
     }
   }, [i18nextInstance, contentLanguage])
-
   // Apply side effects
   useEffect(() => {
     if (document.documentElement) {
       document.documentElement.lang = language
     }
 
-    // Only change ui direction from default ('ltr') if the language is supported
-    if (config.isSupportedLanguage(language)) {
-      dispatch(setUiDirection(config.hasRTLScript(language) ? 'rtl' : 'ltr'))
-    }
-  }, [dispatch, language])
+    // TODO Remove uiDirection from redux state
```

Comment:
I think we will need it in the redux state, but maybe I'm wrong.
I think for this case it would be simpler to keep it in the state

Reply:
I think we don't need it :) Basically we can just pass uiDirection or language as prop and check in each component separately, can't we? If we need redux just for this tiny prop something is wrong imo. But lets just try without and if we need it we can still add it.

Reply:
Is this what you tried already for the tooltips, right? Why did you revert it again? If this is not possible/leads to too many changes, I think e.g. using react context would be the better alternative.